linux/utils/get_network_manager.yml

```diff
@@ -55,7 +55,7 @@
             guest_os_network_manager: "systemd-networkd"
           when: >
             (guest_os_ansible_distribution == "VMware Photon OS") or
-            ("Flatcar" in guest_os_ansible_distribution)
+            ("Flatcar" == guest_os_ansible_distribution)
```

Comment:
This condition can be replaced with `guest_os_ansible_distribution in ['VMware Photon OS', 'Flatcar']`